sonar-ws/src/main/java/org/sonarqube/ws/client/WsClient.java

```diff
@@ -83,6 +117,12 @@
 
   QualitygatesService qualityGates();
 
+  /**
+   * @deprecated since 7.0 use {@link #measures()} instead
+   */
+  @Deprecated
+  org.sonarqube.ws.client.measure.MeasuresService measuresOld();
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)